### PR TITLE
🥸 bugfix/incorrect-voter-stake

### PR DESCRIPTION
**Pull request by stepien4soft**
See: #2685

### CODE REVIEW

File: packages/ui/src/app/pages/Council/Council.tsx [hunk 1]
```diff
@@ -22,9 +23,10 @@ export const Council = () => {
   const { idlePeriodRemaining, budget, reward } = useCouncilStatistics(council?.electedAt.number)
   const { activities } = useCouncilActivities()
   const { stage: electionStage } = useElectionStage()
-
   const [order, setOrder] = useState<CouncilOrder>({ key: 'member' })
   const councilors = useMemo(() => council?.councilors.slice(0).sort(sortBy(order)) ?? [], [council])
+  // const data = useVoteStake( [council.id], [] )
+  // console.log('data', data)
```
Comment (thesan) [inline packages/ui/src/app/pages/Council/Council.tsx:29]:
```suggestion
  const { councilors  } = useCouncilorWithDetails(council)
  const sortedCouncilors = useMemo(() => councilors.sort(sortBy(order)), [councilors])
```

**Review by thesan — COMMENTED** [review]
@stepien4soft I've tested the queries hopefully they'll keep working :crossed_fingers:

**Comment by traumschule** [comment]
LGTM!
- In light of this bug https://github.com/Joystream/joystream/issues/3530#issuecomment-1120361999 it would be very interesting to also see failed candidates somehow - maybe grayed out below or with a heading `Not Elected`?
- Would be great to see stakes for past councils as well (including failed candidates) like here https://dao-git-bugfix-voter-stake-fix-joystream.vercel.app/#/council/past-councils/00000006
- Is it possible to review former candidacies like bullet points, image and program? Basically like Election View during an election but for past terms and with final vote power.

WDYT @dmtrjsg shall i create separate issues?

File: packages/ui/src/council/queries/council.graphql [hunk 2]
```diff
@@ -175,6 +178,23 @@ query GetElectedCouncil {
...
 }
 
+query GetElectionRound {
+  electedCouncils(where: { isResigned_eq: false }, orderBy: [createdAt_DESC], limit: 1) {
+    councilElections {
+      cycleId
+    }
+  }
+}
+
```
Comment (thesan) [inline packages/ui/src/council/queries/council.graphql:188]: AFAICS this query is not used

**Review by thesan — APPROVED** [review]
Looks good :+1: , almost done !
Please just remove the unused query and regenerate the queries before merging

**Pull request merged**